### ISSUE TITLE
Issue in wwivconfig Auto user validation configuration

### ISSUE THREAD

**Issue by MorgulWWIV** [wwivbbs/wwiv#1356]
This is in the linux 5.6.1.

If you go into wwivconfig and select Autovalidation Level Configuration, it opens a window listing the Alt-F# keys on the left.  The window shows Alt-F2 through Alt-F11, but the actual keys are Alt-F1 through Alt-F10.  Clicking on the first entry, Alt-F2, shows that it's actually Alt-F1.  The labels on the first menu are off by one.

Minor issue.

**Comment by wwivsoftwareservices** [comment]
Is this Morgul as in Mr. Dooley?

**Comment by MorgulWWIV** [comment]
It is.

**Comment by fcreid** [comment]
Awesome! :)

**Comment by wwivsoftwareservices** [comment]
The old gang is gathering up I guess.  

 

My renewed interest is based in part on the present day assault on free speech. Been harkening back to the old days where it was promoted and supported on local run systems. We’ve come so far from 1200 bps technology and now everything else is being  slammed into reverse. 

 

Highly doubt we will ever gin up the interest from the highwater WWIV days as we don’t offer the flashiness and appeal to the point and click generation but I’m opening up soon to local folks looking for a secure platform.   Scary times…..

**Comment by wwiv** [comment]
I think this is fixed in 5.6.1.3406 (https://github.com/wwivbbs/wwiv/commit/9a5f1062ce00b69fe5967a229b0112fd2684cb1b), we've had a few fixes since the preview release.  Once thing end up in a steady state, 5.6.1 (hopefully) will be declared GA and we'll move on to 5.7

If we find a few high priority fixes on top of 5.6 (I think we did that once in the past), we can make a 5.6.2 release with those.

**Comment by wwiv** [comment]
Forgot to say, thanks for the issue!  Also great to see you on here @MorgulClan.  Love any feedback you have on how I did FTN support, I suspect you still remember it better than I ever learned it :)

**Comment by MorgulWWIV** [comment]
Ok, good to know.  I'm running 3384 right now.

Haven't looked at FTN yet.  But it's been a LONG time since I looked at that.  My coding skills are very rusty.  I did find my WWIVTOSS document and was reading over it.  I seemed like I knew what I was doing back then!

**Comment by wwivsoftwareservices** [comment]
Doubt very seriously if I will get into the coding at all. I’ll do some python these days but I have not done C, C++, or C# in years. Really don’t see myself getting back into it.

================
Sent from a mobile device. Please excuse brevity, shorthand, or errors in grammar and spelling.
________________________________

**Comment by MarkHofmann11** [comment]
I still use WWIVTOSS over here - I did make a few modifications to it a few
years ago.  One was doing an rtrim on the Username, as FTN user's were not
seeing matches to my Username on their BBS.  There was a hidden space at
the end of the usernames that I had to use rtrim to remove.  Very minor
updates, but it still works!!! :)

These days I have done some Python mods mainly with Home Assistant.  Not
much else when it comes to that.  I'm a big hardware junkie and storage
hoarder, though.
